src/Polysemy/Internal/Combinators.hs

```diff
@@ -80,12 +80,11 @@ interpretH
        -- already in 'Sem'.
     -> Sem (e ': r) a
     -> Sem r a
-interpretH f (Sem m) = m $ \u ->
+interpretH f (Sem m) = Sem $ \k -> m $ \u ->
```

Comment:
just out of curiosity, why did you change that?

Reply:
Running a `Sem` as the universally quantified monad `m` inside of another `Sem` has always been more efficient than running a `Sem` as a `Sem` directly, so I figured I'd do that optimization while I was at it.

Reply:
so adding more stuff makes it go faster? :grin: